FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/History.txt

```diff
@@ -1,3 +1,47 @@
+Changes between V2.2.1 and V2.2.2
+	+ Add CBMC proofs for 4 functions.
+	+ Address ~1200 MISRA violations.
+	+ Support caching multiple IP addresses per DNS entry in TCP library.
+	+ Security improvements in the TCP library.
+	+ Add additional length check when checksum offloading is used.
```

Comment:
"Improve robustness by checking packet lengths in software even if the check has already been performed in the hardware".